Joomla/Sniffs/Commenting/FunctionCommentSniff.php

```diff
@@ -231,7 +235,8 @@ protected function processParams(PHP_CodeSniffer_File $phpcsFile, $stackPtr, $co
 						{
 							if ($tokens[$i]['code'] === T_DOC_COMMENT_STRING)
 							{
-								$comment .= ' ' . $tokens[$i]['content'];
+								$comment   .= ' ' . $tokens[$i]['content'];
+                                $commentEnd = $i;
```

Comment:
Ugh, Stupid github web interface and their enforced tabs=8 spacing making it hard to see things

Should be fixed...
(although I'm not sure why that didn't flag in the rules check... maybe something isn't working or we need another sniff for that.)